external/loaders/loaders/mappers/_fine_res.py

```diff
@@ -9,6 +9,7 @@
 import fsspec
 
 from vcm.fv3.metadata import gfdl_to_standard
+from vcm.limit import DatasetQuantileLimiter
```

Comment:
Limiting is a post-processing feature that could be applied to any mapper, so it's odd to couple it to just these few mappers. Ideally mappers would just be simple data objects...and any downstream tools (e.g. prog run or fv3fit) can use `vcm.DatasetQuantileLimiter` in their own pre/post processing pipelines.

For now, you should move this logic to the prognostic run...which I think is the only place you currently plan to use it now.

Reply:
Good suggestion, I moved it to the tendency prescriber